src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -692,9 +692,10 @@ namespace DataHeatBalance {
         Real64 delta_T = 0.0;                               // Indoor and outdoor temperature
         Real64 delta_HumRat = 0.0;                          // Indoor and outdoor humidity ratio delta
 
-        Real64 ZeroSourceSumHATsurf = 0.0; // From Chilled Ceiling Panel, equal to the SumHATsurf for all the walls in a zone with no source
-        bool zoneOAQuadratureSum = false;  // True when zone OA balance method is Quadrature
-        int zoneOABalanceIndex = 0;        // Index to ZoneAirBalance for this zone, if any
+        Real64 ZeroSourceSumHATsurf = 0.0;   // From Chilled Ceiling Panel, equal to the SumHATsurf for all the walls in a zone with no source
+        bool zoneOAQuadratureSum = false;    // True when zone OA balance method is Quadrature
+        int zoneOABalanceIndex = 0;          // Index to ZoneAirBalance for this zone, if any
+        Real64 ZeroBBSourceSumHATsurf = 0.0; // used in baseboards in energy balance
```

Comment:
And let's not forget about that other ZeroSourceSumHATsurf variable here that is used for chilled ceiling panels. Maybe this variable should also be moved over to the ChilledCeilingPanelSimple? Moving these variables around while cleaning up code does scare me a little.